Make `webcomponents-loader.js` compatible with the Trusted Types API (#501)
webcomponents/polyfills: packages/webcomponentsjs/README.md

<script type="module"></script>
```

#### Trusted Types

If you're using the loader on a page that enforces the `trusted-types` Content
Security Policy, you'll need to allow the `webcomponents-loader` policy name so
that the loader can dynamically create and insert a `<script>` for the polyfill

> bundle it selects based on feature detection. You should also treat
> `WebComponents.root` as if it were required to be set to a `TrustedScriptURL`
> when trusted types are being enforced.

Reviewer: ```suggestion
bundle it selects based on feature detection. If you set `WebComponents.root`
(which is rare), it should be set to a TrustedScriptURL for Trusted Types
compatibility.
```

Author: I'll work on the language more here. I don't think we can actually enforce that this is a `TrustedScriptURL` without making a breaking change, but I think I could word this so that the idea that it should only be set to values that would be considered as safe as a `TrustedScriptURL` was more clear. I.e. we can't actually enforce the safety ourselves, so users should know that it needs to be treated this way.

Author: I've updated this slightly, PTAL

Reviewer: WDYT of my recommended wording here?

Author: Yeah, this is better - updated.
